website/widgets/form/table.md

```diff
@@ -352,3 +352,24 @@ q.page['example'] = ui.form_card(box='1 1 3 6', items=[
 ```
 
 Check the [complete example](/docs/examples/table-pagination) with all the supported features like sort/filter/search/download/reset.
+
+## With links
+
+Links can be displayed by setting `cell_type` to `'ui.url_table_cell_type'`. If `'new_tab'` is set to `'True'` the link will be opened on a new tab.
```

Comment:
Do not forget to rename the `new_tab` attribute.

Reply:
Overlooked. Renamed.